### ISSUE TITLE
TypeError after upgrade to 0.2.2

### ISSUE THREAD

**Issue by IngmarStein** [dubocr/homebridge-tahoma#95]
Hi,

after upgrading to 0.2.2, my homebridge service (with minimal config for homebridge-tahoma) crashloops with the following backtrace:
```
[5/9/2019, 11:04:31 PM] TypeError: Cannot read property 'length' of null
    at Server.<anonymous> (/usr/local/lib/node_modules/homebridge/lib/server.js:407:44)
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/lib/util/once.js:16:19
    at /homebridge/node_modules/homebridge-tahoma/index.js:96:6
    at TahomaPlatform.<anonymous> (/homebridge/node_modules/homebridge-tahoma/index.js:190:5)
    at /homebridge/node_modules/homebridge-tahoma/overkiz-api.js:184:5
    at Request._callback (/homebridge/node_modules/homebridge-tahoma/overkiz-api.js:249:6)
    at Request.self.callback (/homebridge/node_modules/request/request.js:185:22)
    at Request.emit (events.js:189:13)
    at Request.<anonymous> (/homebridge/node_modules/request/request.js:1161:10)
    at Request.emit (events.js:189:13)
    at IncomingMessage.<anonymous> (/homebridge/node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:277:13)
    at IncomingMessage.emit (events.js:194:15)
    at endReadableNT (_stream_readable.js:1125:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)
```

**Comment by IngmarStein** [comment]
The above might be a result of the login call being rate-limited due to the crashloop. The actual error might be this one:
```
[5/9/2019, 11:04:42 PM] [TaHoma] Instanciate generic Sonos as Switch
[5/9/2019, 11:04:43 PM] ReferenceError: Service is not defined
    at new ScenarioAccessory (/homebridge/node_modules/homebridge-tahoma/index.js:226:17)
    at /homebridge/node_modules/homebridge-tahoma/index.js:200:32
    at /homebridge/node_modules/homebridge-tahoma/overkiz-api.js:193:5
    at Request.authCallback [as _callback] (/homebridge/node_modules/homebridge-tahoma/overkiz-api.js:216:17)
    at Request.self.callback (/homebridge/node_modules/request/request.js:185:22)
    at Request.emit (events.js:189:13)
    at Request.<anonymous> (/homebridge/node_modules/request/request.js:1161:10)
    at Request.emit (events.js:189:13)
    at IncomingMessage.<anonymous> (/homebridge/node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:277:13)
    at IncomingMessage.emit (events.js:194:15)
    at endReadableNT (_stream_readable.js:1125:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)
```

**Comment by IngmarStein** [comment]
The crashloop stops when flipping `exposeScenarios` to `false` (which worked previously).

**Comment by dubocr** [comment]
Hi,
Must be fixed in 0.2.3
Keep me informed.
Thks

**Comment by IngmarStein** [comment]
All good again, thanks for the quick fix!
